### PR TITLE
public docs for oci logging analytics

### PR DESCRIPTION
**Pull request by adiforluls**
This PR is for documentation of Oracle Cloud Infrastructure Logging Analytics output plugin.
Corresponds to https://github.com/fluent/fluent-bit/pull/7830

### CODE REVIEW

**Comment by adiforluls** [comment]
cc: @edsiper @patrick-stephens

File: pipeline/outputs/oci-logging-analytics.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
+---
+description: Send logs to OCI Logging Analytics Service
+---
+
+# OCI Logging Analytics
```
Comment (patrick-stephens) [inline pipeline/outputs/oci-logging-analytics.md:5]: We do not introduce the acronym `OCI` until later, personally I think we should be clear this is Oracle Cloud Infrastructure Logging and nothing to do with the Open Container Initiative which is what I would expect of that acronym.

Can you update the description and title to use the full name?

File: pipeline/outputs/oci-logging-analytics.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+## TLS/SSL
+
+OCI Logging Analytics output plugin supports TTL/SSL, for more details about the properties available and general configuration, please refer to the [TLS/SSL](../../administration/transport-security.md) section.
+
+## Getting Started
+
+### Prerequisites
+
+- OCI Logging Analytics service must be onboarded with the minumum required policies, in the OCI region where you want to monitor. Refer [Logging Analytics Quick Start](https://docs.oracle.com/en-us/iaas/logging-analytics/doc/quick-start.html) for details.
```
Comment (patrick-stephens) [inline pipeline/outputs/oci-logging-analytics.md:45]: Do we need to do anything in the FB config or set up env vars, etc.?

Linking out is useful for context but we should have all config here.

File: pipeline/outputs/oci-logging-analytics.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+- Create OCI Logging Analytics LogGroup(s) if not done already. Refer [Create Log Group](https://docs.oracle.com/en-us/iaas/logging-analytics/doc/create-logging-analytics-resources.html#GUID-D1758CFB-861F-420D-B12F-34D1CC5E3E0E) for details.
```
Comment (patrick-stephens) [inline pipeline/outputs/oci-logging-analytics.md:46]: Is this link stable? Just a little concerned with the need for a GUID.

**Review by patrick-stephens — COMMENTED** [review]
Only a few niggles.

How would you suspect people would use this? Wondering if we should expand the examples away from just dummy input to real scenarios?

**Comment by patrick-stephens** [comment]
Please also update the description and link back to the original PR in the code.

File: pipeline/outputs/oci-logging-analytics.md (hunk 1 above)
Comment (adiforluls) [reply pipeline/outputs/oci-logging-analytics.md]: Fixed

File: pipeline/outputs/oci-logging-analytics.md (hunk 2 above)
Comment (adiforluls) [reply pipeline/outputs/oci-logging-analytics.md:45]: No, we don't need anything in the fb config or setup env vars as part of setting up logging analytics. The configs required by fluent-bit have been documented in this doc.

File: pipeline/outputs/oci-logging-analytics.md (hunk 3 above)
Comment (adiforluls) [reply pipeline/outputs/oci-logging-analytics.md:46]: This is a stable link @patrick-stephens

**Review by patrick-stephens — CHANGES_REQUESTED** [review]
Please update the description and link back to the original code change PR too.

**Comment by adiforluls** [comment]
Done @patrick-stephens

**Review by patrick-stephens — APPROVED** [review]

**Comment by adiforluls** [comment]
Hi @patrick-stephens , the main code is merged. This PR can be merged now

**Comment by patrick-stephens** [comment]
> Hi @patrick-stephens , the main code is merged. This PR can be merged now

Not until it's in a release usually, otherwise it'll confuse people expecting to use it with the official binaries.

**Pull request merged**